### ISSUE TITLE
Presence of target domain labels 

### ISSUE THREAD

**Issue by sparshgarg23** [agrija9/Deep-Unsupervised-Domain-Adaptation#5]
I have noticed that most of the UDA experiments on Office 31 or visda provide a train_list.txt and val_list.txt during the dataloader creation stage.
Doesn't providing the list for both source and target domain defeat the purpose of  unsupervised domain adaptation where target domain has no information about the target labels.

I was looking at the organization of office 31 dataset as well as the dataloader part in your code,and i feel that during training based on how data is being loaded ,the model will have access to the real target domain labels as well.
Of course,if there is something which I am missing here please correct me.

**Comment by BaeHann** [comment]
Within the CDAN function, entropy.detach() was not found. Does that mean the entropy will be affected (as well as the classification result and relevant parameters in model) when loss.backward() was executed?
